tests/testthat/helper-for-tests.R

```diff
@@ -0,0 +1,11 @@
+loadTestSimulation <- function(simulationName) {

```

Comment:
@PavelBal It's possible to define helper for tests which is awesome to avoid code repetition